themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml

```diff
@@ -45,7 +49,7 @@
   <?php endif; ?>
 
   <?php if (!$this->facet['isApplied'] && null !== $this->facet['count']): ?>
-    <span class="badge">
+    <span class="badge" aria-label="<?=$this->transEsc('results') ?>:">
```

Comment:
Might it be more clear to add a new label for "Result count:"?

Reply:
I don't quite understand what you mean -- this is exactly what I had?

Reply:
Maybe I'm misunderstanding something, but as currently written, it's just going to label the block with the word "results". Won't that read out like "results 150"? That's not going to be grammatically correct in most languages, so I thought it might be less confusing if we made the label more like a label, so it reads out as "Result count: 150". But maybe I'm mistaken about how the aria-label attribute is applied in this context.

Reply:
good point, I didn't consider other languages ... I'd stick with "results" because we already have       
`result_count = "%%count%% Treffer"` but changed the order

Reply:
Done in the comment below